packages/cli/src/commands/serve.ts

```diff
@@ -13,6 +13,7 @@ export default class Serve extends Command {
     spec: oflags.string({
```

Comment:
Do we want to have required options? Is this one maybe something that should just be an argument? 

If you are trying to serve a think it seems like you are gonna definitely give it a spec file? Are there any other required options?